src/utils/confirm.js

```diff
@@ -0,0 +1,7 @@
+import { usePopup } from 'hooks'
+import { ConfirmDialog } from 'components'
+
+export default function ({ message, onSubmit }) {
```

Comment:
Is there a strong reason to have a different style here? All the components are named export of a lambda and this is a default export of a function.

Reply:
true, this just returns a single function.